setup/gcp/installer/util/gcloud.py

```diff
@@ -626,6 +631,14 @@ def get_vm_instance_info(instance_name, try_match=False):
         str: Zone of the forseti server application, default to 'us-central1-c'
         str: Name of the forseti server instance
     """
+    def _ping_compute_instance():
+        """Check compute instance status.
+        """
+        utils.run_command(
+            ['gcloud', 'compute', 'instances', 'list', '--format=json'])
```

Comment:
Future PR: Let's also use --verbosity in all gcloud commands and replace it with what the user runs the installer as, e.g.

`python setup/installer.py --verbosity=DEBUG` then `['cloud', 'compute', 'instances', 'list', '--format=json', '--verbosity={}'.format(LOG_LEVEL)]`

Reply:
Opened issue #1033 to keep track of this.